extensions/wikia/TrackingOptIn/js/trackingOptIn.js

```diff
@@ -19,7 +19,15 @@ define('wikia.trackingOptIn', [
 	function init() {
 		if (instantGlobals.wgEnableTrackingOptInModal) {
 			log('Using tracking opt in modal', log.levels.info, logGroup);
-			var instance = trackingOptInModal.init({
+
+			var usapiEnabled =
+				window &&
+				window.location &&
+				window.location.search &&
+				window.location.search.indexOf('icUSPrivacyApi=1') !== -1;
+
+			var instances = trackingOptInModal.init({
+				enableCCPAinit: usapiEnabled,
```

Comment:
A thought for later: now it's only California enforcing the law but the mechanism is called "us privacy api". Maybe we should one day use that name like "usp" instead of ccpa?

Reply:
This variable is temporary. Once we'll finish tests we'll make this global and remove it. US Privacy is a official name for that mechanism.